### PR TITLE
chore: Taxonomy change requests week 40

### PR DESCRIPTION
**Pull request by aleene**
### What
The taxonomy change requests that have been submitted in week 40 on Slack and elsewhere

### CODE REVIEW

**Review by stephanegigandet — APPROVED** [review]
Looks good but:

the image:en: property is not used

currently we only display logos with an height of 90px.

**Comment by aleene** [comment]
@stephanegigandet Can we define a process to add images and link them with a taxonomy?

**Comment by stephanegigandet** [comment]
> @stephanegigandet Can we define a process to add images and link them with a taxonomy?

@aleene I filed https://github.com/openfoodfacts/openfoodfacts-server/issues/7447

File: taxonomies/labels.txt [hunk 1]
```diff
@@ -19142,6 +19147,14 @@ origins:en: en:france
 label_categories:en: en:Honey
 ingredients:en: en:honey
 
+#<en:supply chain labels
+#<en:certified labels
+description:en:True Source Honey is a voluntary system of traceability and authenticity testing for those participants who wish to demonstrate through an independent third party that their sourcing practices are in full compliance with U.S. and international trade laws.
+en:True Source Certified
```
Comment (stephanegigandet) [inline taxonomies/labels.txt:19153]: This line must be the first line, we can't have properties like description:en: above it.

**Pull request merged**